README.md

```diff
@@ -8,41 +8,35 @@ This project serves as the inventory source for the [Ansible Tower Configuration
+The projects `CREDENTIAL_CONFIG.yml` file contains a list of variables which need to be changed to suit your own tower environment. Once these variables have been changed,the `bootstrap.yml` playbook consumes the `CREDENTIAL_CONFIG.yml` file, encrypts any variables containing sensitive information using the supplied Ansible vault password, and then places each of the variables into the relevant group_vars file. The `CREDENTIAL_CONFIG.yml` file itself is then encrypted using the same Ansible vault password.
 
 ## Setup
 
 **NOTE**: This project should be forked and made private before proceeding with the following setup steps.
 
-1. Clone this project
+1. Clone this project.
 
 ```bash
 git clone https://github.com/<forked_repository_org>/tower_dummy_credentials
 ```
 
-2. Copy and rename the `SAMPLE_CREDENTIAL_CONFIG.yml` file locally. As the local copy of the config will sensitive information in plaintext, it should only be stored and referenced locally, and caution taken to not check it into a public repository.
+1. Change the variable values in the `CREDENTIALS_CONFIG.yml` file. A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
 
-```bash
-cp SAMPLE_CREDENTIAL_CONFIG.yml local_credentials_config.yml
-```
-
-3. Change the variable values in the newly created local copy of the `local_credentials_config.yml` file. A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
-
-4. From the projects root directory, run the `bootstrap.yml` playbook, specifying the path to your local copy of the credentials file.
+2. From the projects root directory, run the `bootstrap.yml` playbook, specifying the path to the `CREDENTIALS_CONFIG.yml` file.
 
 ```bash
-ansible-playbook -i ./inventories/hosts bootstrap.yml --extra-vars='@local_credentials_config.yml'
+ansible-playbook -i ./inventories/hosts bootstrap.yml --extra-vars='@CREDENTIALS_CONFIG.yml'
 ```
 
 ## Adding new variables
 
-The following steps outline the process of adding new variables to the project.
+The following steps outline the process of adding new variables to the project. Please ensure that all new variables are also added to the [VARIABLES.md](VARIABLES.md) file.
 
 ### Encrypted
 
-1. Add the variable to the  `SAMPLE_CREDENTIAL_CONFIG.yml` file with a value of `<CHANGEME>`.
+1. Add the variable to the  `CREDENTIAL_CONFIG.yml` file with a default value of `<CHANGEME>`.
 
-2. Add an entry within the `with_items` section of the [bootstrap_credentials.yml](roles/credentials/tasks/bootstrap_credentials.yml#L13-L39) file, replacing `new_variable` with the name of the new variable.
+2. Add an entry within the `with_items` section of the [bootstrap_credentials.yml](roles/credentials/tasks/bootstrap_credentials.yml#L13-L39) file, replacing `new_variable` in the both the name and value fields with the name of the new variable.
```

Comment:
The line numbers in the linked URL will change depending on the number of variables that get added so probably best just to point to the first line of the with_items section: https://github.com/integr8ly/tower_dummy_credentials/blob/ca36707d6dfa522ff10057939d17f64f8c93605e/roles/credentials/tasks/bootstrap_credentials.yml#L13